ruby/secret_agents.rb

```diff
@@ -0,0 +1,34 @@
+def encrypt(text)
```

Comment:
Both of these methods look good though there is a minor take away and a big take away.

minor take away - style wise, add a space before and after your '='.

major take away - methods have return values which are dictated by the last evaluated expression in our method or when the keyword `return` is used.  In our case the last evaluated expression is `puts new_string`.  What does this return?
How does this effect the return of our method encrypt?
If you are unsure, try and figure out the return of your method encrypt - hint, it is not a string.